pkg/config/config.go

```diff
@@ -58,25 +58,26 @@ var (
 )
 
 type Config struct {
-	Port           uint32                   `yaml:"port,omitempty"`
-	BindAddresses  []string                 `yaml:"bind_addresses,omitempty"`
-	PrometheusPort uint32                   `yaml:"prometheus_port,omitempty"`
```

Comment:
Can we keep the field and mark it as deprecated for now? We'd like to avoid breaking existing users who's activated Prometheus.

If the old value is set, perhaps we can print a warning and copy that value to the new field.